bin/Snack.hs

```diff
@@ -460,8 +467,8 @@ runCommand :: SnackConfig -> PackageFile -> Command -> IO ()
 runCommand snackCfg packageFile = \case
   Build -> S.shelly $ void $ snackBuild snackCfg packageFile
   Run args -> quiet (snackBuild snackCfg packageFile) >>= runBuildResult args
-  Ghci -> flip runExe [] =<<
-    ghciExePath <$> (quiet (snackGhci snackCfg packageFile))
+  Ghci args -> flip runExe [] =<<
+    ghciExePath <$> (quiet (snackGhci snackCfg packageFile args))
```

Comment:
instead of passing the arguments through the nix files, why not just passing them to `runExe` and have the (wrapped) ghci accept extra CLI arguments? this way you wouldn't have to thread `ghciOptions` throughout `snack-lib`. Does that make sense? i.e. `ghciWithModules` would look like this:

> ${ghc}/bin/ghci ${lib.strings.escapeShellArgs ghciArgs} $@

Reply:
Oops.  I did not know I can pass args like that.  I have just fixed that.